tests/api_test/api_test.cpp

```diff
@@ -288,8 +288,24 @@ ring_buffer_api_test(ebpf_execution_type_t execution_type)
     });
 }
 
+void
+divide_by_zero_test(ebpf_execution_type_t execution_type)
```

Comment:
This name and prototype look identical to divide_by_zero_test in end_to_end.cpp.   What is the difference?   At least add // comments to both places explaining what the difference is from the other one?

Reply:
I see you renamed it to end in _km which is ok (and I assume that means yes the difference is just UM vs KM), but would still like better comments that cross reference the two places.  E.g., rename the other one to end in _um and add // comments above both of them such as:
```
api_test.cpp:
// See also divide_by_zero_test_um in end_to_end.cpp for the user-mode equivalent.
void
divide_by_zero_test_km(ebpf_execution_type_t execution_type)
...
end_to_end.cpp:
// See also divide_by_zero_test_km in api_test.cpp for the kernel-mode equivalent.
void
divide_by_zero_test_um(ebpf_execution_type_t execution_type)
```